components/organisms/ThemeContext/ThemeProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { createContext, useContext, useState } from 'react';
+
+const ThemeContext = createContext();
+
+export const ThemeProvider = ({ children, value }) => {
+    const [theme, setTheme] = useState(value);
```

Comment:
jak stworzysz enuma Theme, to zapisz tutaj

`const [theme, setTheme] = useState<ThemeEnum>(value);`